core/domain/takeout_service_test.py

```diff
@@ -758,6 +761,38 @@ def set_up_non_trivial(self):
         report_entity.update_timestamps()
         report_entity.put()
 
+        # Set-up for the BlogPostModel.
+        blog_post_model = blog_post_models.BlogPostModel(
+            id=self.BLOG_POST_ID_1,
+            author_id=self.USER_ID_1,
+            content='content sample',
+            title='sample title',
+            published_on=datetime.datetime.utcnow(),
+            url_fragment='sample-url-fragment',
+            tags=['tag', 'one'],
+            thumbnail_filename='thumbnail'
+        )
+        blog_post_model.update_timestamps()
+        blog_post_model.put()
+
+        blog_post_rights_old = blog_post_models.BlogPostRightsModel(
```

Comment:
I don't quite follow this ... what are the semantics of "old" and "new" here?

Reply:
Just to differentiate between two models. Should I rename them as `blog_post_rights_model_one` and `blog_post_rights_model_two`

Reply:
Try naming them in a way that makes it easy to tell what they are, e.g. blog_post_rights_for_post_1, etc.

Reply:
Done.